modules/get-started/pages/intro-to-rpk.adoc

```diff
@@ -64,7 +57,9 @@ Or, get a detailed description about each option with `-X help`:
 rpk -X help
 ----
 
-Some of the common configuration properties apply across all `rpk` commands as defaults. These default properties have keys with names starting with `defaults`, and they're viewable in `rpk -X list` and `rpk -X help`. For more details, see xref:reference:rpk/rpk-x-options.adoc[`rpk -X options`].
+Every `-X` option can be translated into an environment variable by prefixing with `RPK_` and replacing periods (`.`) with underscores (`_`). For example, the flag `tls.enabled` has the equivalent environment variable `RPK_TLS_ENABLED`.
```

Comment:
This paragraph is a copy from https://docs.redpanda.com/current/reference/rpk/rpk-x-options/